src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1731,14 +1733,6 @@ public void updateTriggerConfigURL() {
             String triggerInformationMessage = MessageFormat.format(
                     "ParseException when fetching dynamic trigger url: {0}", pe.getMessage());
             triggerInformationAction.setErrorMessage(triggerInformationMessage);
-        } catch (MalformedURLException mue) {
-            String logErrorMessage = MessageFormat.format(
-                    "MalformedURLException for project: {0} and URL: {1} Message: {2}",
```

Comment:
Why is this removed? The user could still specify a malformed url in the config screen, and now the error will be harder to find.

Reply:
Agree, will restore.